src/test/scala/uk/gov/hmrc/api/client/HttpClient.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.api.client
+
+import org.apache.pekko.actor.ActorSystem
+import play.api.libs.ws.*
+import play.api.libs.ws.DefaultBodyWritables.*
+import play.api.libs.ws.ahc.StandaloneAhcWSClient
+
+import scala.concurrent.{ExecutionContext, Future}
+
+trait HttpClient {
```

Comment:
This looks like it's using the old HMRC Http Client. 

https://github.com/hmrc/http-verbs is the new version